modules/ROOT/pages/tools/cypher-shell.adoc

```diff
@@ -262,7 +262,7 @@ You can use the `:source` command followed by the file name to run the Cypher st
 
 [source, shell]
 ----
-:source example.cypher
+:source /tmp/example.cypher
```

Comment:
I would suggest that we put something more general, as I am not sure if everyone puts their Cypher scripts in the /tmp folder.
```suggestion
:source /path/to/your/example.cypher
```

Reply:
Thank you! It's a good idea.